apps/backend/src/routes/chip/attendance.ts

```diff
@@ -0,0 +1,60 @@
+import express, {Request, Response} from "express";
+import {Controller} from "@/lib/controller";
+import {
+  ChipAttendanceRequest,
+  ErrorResponse,
+  ChipAttendanceRequestSchema
+} from "@types";
+
+const router = express.Router();
+const controller = new Controller();
+
+/**
+ * @route GET /api/chip/:id/attendance
```

Comment:
I could also do a user lookup instead -- ie from user, get chip and from chip get attendance. I used chip because there were fewer lookup steps required.